appveyor.yml

```diff
@@ -12,12 +12,16 @@ install:
   - rustup-init.exe --default-host "%ARCH%-pc-windows-gnu" --default-toolchain %RUST% -y
   - SET PATH=C:\Users\appveyor\.cargo\bin;C:\msys64\mingw%BITS%\bin;%PATH%;C:\msys64\usr\bin
   - SET LIBGIT2_SYS_USE_PKG_CONFIG=1
+  - IF "%RUST%" == "stable" rustup component add rustfmt clippy
   - rustc -Vv
   - cargo -Vv
   - pacman --noconfirm -S mingw-w64-%ARCH%-gtk3 mingw-w64-%ARCH%-libgit2 mingw-w64-%ARCH%-libsecret
 
 build_script:
   - git clone -q https://github.com/gtk-rs/gir-files tests/gir-files
+  - IF "%RUST%" == "stable" cargo clippy --all --release
+  - IF "%RUST%" == "stable" cargo check --all --release
```

Comment:
apveyor check only stables, so `if` don't needed,
You sure that `cargo check` can show something new after `cargo clippy` ?
Why is `--all` used, gir don't have workspaces.

Reply:
No, I'm not sure. `--all` is from copying the invocations from somewhere else.